### ISSUE TITLE
multiple package manager lockfiles in web3.js

### ISSUE THREAD

**Issue by wentokay** [solana-labs/solana#25148]
#### Problem

There's a `yarn.lock` and a `package-lock.json` in web3.js

#### Proposed Solution

Pick one package manager for the project and remove the lockfile that's associated with the other one

**Comment by jstarry** [comment]
Yeah, @steveluscher why do we need both?

**Comment by steveluscher** [comment]
Yeah! I noticed that. I'm not sure when the `npm` one snuck in, but it should begone.

**Comment by jstarry** [comment]
`package-lock.json` was always in there, I noticed you added the additional `yarn.lock` file. I prefer `npm` personally, any reason you use `yarn`?

**Comment by steveluscher** [comment]
`npm` has gotten a lot better, but I still carry damage from the days that made the creation of `yarn` necessary.

**Comment by steveluscher** [comment]
Oops. Our build cache still relies on `package-lock.json` so it wasn't as simple as deleting it. I'll have to revisit this in a hot moment after I revert the deletion.
